### PR TITLE
[Docs] Update the WasmEdge-Go doc to 0.11.0.

### PR DESCRIPTION
**Pull request by q82419**
Signed-off-by: YiYing He <yiying@secondstate.io>

### CODE REVIEW

File: docs/book/en/src/sdk/go/0.9.1/ref.md [hunk 1]
```diff
@@ -1219,6 +1313,123 @@ WasmEdge VM provides APIs for developers to register and export any WASM modules
...
+      }
+      async.Release()
+      vm.Release()
+    }
+    ```
+
+    Then you can build and run: (the 25th Fibonacci number is 121393 in 0-based index)
+
+    ```bash
+    $ go get github.com/second-state/WasmEdge-go/wasmedge@v0.10.1
```
Comment (hydai) [inline docs/book/en/src/sdk/go/0.9.1/ref.md:1427]: This file is 0.9.1

File: docs/book/en/src/sdk/go/0.9.1/ref.md [hunk 2]
```diff
@@ -1219,6 +1313,123 @@ WasmEdge VM provides APIs for developers to register and export any WASM modules
...
+      }
+      async.Release()
+      vm.Release()
+    }
+    ```
+
+    Then you can build and run: (the 20th Fibonacci number is 10946 in 0-based index)
+
+    ```bash
+    $ go get github.com/second-state/WasmEdge-go/wasmedge@v0.10.1
```
Comment (hydai) [inline docs/book/en/src/sdk/go/0.9.1/ref.md:1360]: This file is 0.9.1

File: docs/book/en/src/sdk/go/0.9.1/ref.md [hunk 3]
```diff
@@ -23,13 +23,16 @@ The followings are the guides to working with the WasmEdge-Go SDK at WasmEdge ve
   * [Results](#results)
   * [Contexts And Their Life Cycles](#contexts-and-their-life-cycles)
   * [WASM data structures](#wasm-data-structures)
+  * [Async](#async)
```
Comment (hydai) [inline docs/book/en/src/sdk/go/0.9.1/ref.md:26]: Not pretty sure why do we need to update the 0.9.1 document?

**Review by hydai — CHANGES_REQUESTED** [review]

File: docs/book/en/src/sdk/go/0.9.1/ref.md (hunk 3 above)
Comment (q82419) [reply docs/book/en/src/sdk/go/0.9.1/ref.md:26]: The docs of v0.9.0 lost the implemented APIs.

**Review by hydai — APPROVED** [review]

**Pull request merged**